docs/build-android.md

```diff
@@ -33,6 +33,12 @@ This will take quite some time depending on the speed of your internet connectio
 
 ## Build Android
 
+Initialize the environment with the ```envsetup.sh``` script.
```

Comment:
Please fold the actual into the next paragraph, right before the lunch invocation.